test/08_string.jl

```diff
@@ -70,19 +82,24 @@ using CategoricalArrays
     @test normalize_string(v2, :NFKD) == "café"
 
     @test isempty(collect(graphemes(v1)))
-    @test collect(graphemes(v2)) == collect(graphemes("café"))
+    if VERSION < v"0.7.0-DEV.2949"
+        # FIXME: This causes a StackOverflowError after the string overhaul
+        @test collect(graphemes(v2)) == collect(graphemes("café"))
+    end
 
     @test isvalid(v1)
     @test isvalid(v2)
     @test !isvalid(v1, 1)
     @test isvalid(v2, 4)
     @test !isvalid(v2, 5)
 
-    @test_throws BoundsError ind2chr(v1, 0)
-    @test ind2chr(v2, 4) === 4
+    if VERSION < v"0.7.0-DEV.2949"
+        @test_throws BoundsError ind2chr(v1, 0)
```

Comment:
Shouldn't this be converted to `length` calls?

Reply:
Is there Compat for multi-arg `length`?